<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Function</class>
 <widget class="QWidget" name="Function">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="openPushButton">
         <property name="text">
          <string>File Open</string>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QToolButton" name="zoomInToolButton">
           <property name="text">
            <string>확대</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../Users/KOSA/Desktop/zoomIN.png</normaloff>../../../Users/KOSA/Desktop/zoomIN.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="zoomOutToolButton">
           <property name="text">
            <string>축소</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../Users/KOSA/Desktop/ZoomOut.png</normaloff>../../../Users/KOSA/Desktop/ZoomOut.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QToolButton" name="leftRotateToolButton">
           <property name="text">
            <string>좌회전</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../Users/KOSA/Desktop/leftRotate.png</normaloff>../../../Users/KOSA/Desktop/leftRotate.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="rightRotateToolButton">
           <property name="text">
            <string>우회전</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../Users/KOSA/Desktop/rightRotate.png</normaloff>../../../Users/KOSA/Desktop/rightRotate.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sizePushButton">
         <property name="text">
          <string>Original Size</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView2"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
